src/core_plugins/kibana/public/dashboard/dashboard_strings.js

```diff
@@ -28,10 +29,26 @@ import { DashboardViewMode } from './dashboard_view_mode';
  */
 export function getDashboardTitle(title, viewMode, isDirty) {
   const isEditMode = viewMode === DashboardViewMode.EDIT;
-  const unsavedSuffix = isEditMode && isDirty
-    ? ' (unsaved)'
-    : '';
+  let displayTitle;
 
-  const displayTitle = `${title}${unsavedSuffix}`;
-  return isEditMode ? 'Editing ' + displayTitle : displayTitle;
+  if (isEditMode && isDirty) {
+    displayTitle = i18n.translate('kbn.dashboard.strings.dashboardUnsavedEditTitle', {
+      defaultMessage: 'Editing {title} (unsaved)',
+      values: { title },
+    });
+  } else if (isEditMode && isDirty) {
```

Comment:
The same condition as in 34 line. Please look at my previous comment for this file.